### ISSUE TITLE
Allow showing file list widget instead of file chooser

### ISSUE THREAD

**Issue by imagejan** [scijava/scijava-ui-swing#69]
When using a single `File[]` parameter like:

```
#@ File[] (style="extensions:tif/tiff/ims") files
```

.. we always show a file chooser dialog using `chooseFiles`.
Only when a second unrelated parameter is added, a file list widget is shown in a generated input dialog:

```
#@ File[] (style="extensions:tif/tiff/ims") files
#@ String (required=false) ignoreThis
```

There should be a possiblity to get an input dialog with the file list widget, without the need to add secondary parameters.

(I thought I had opened a ticket about this long ago, but I couldn't find it, so here it is.)

**Comment by ctrueden** [comment]
Yeah, something like `autofill=false`, but for the `FilePreprocessor` and `FileListPreprocessor` to know they shouldn't do this shortcut of popping the file chooser dialog directly. I also remember you mentioning this before, @imagejan, but don't have time to dig for existing issues right now, sorry.
